Makefile.PL

```diff
@@ -2,67 +2,102 @@ use warnings;
 use strict;
 
 use 5.008001;
-BEGIN { unshift @INC, '.' unless $INC[-1] eq '.'; }
-use inc::Module::Install 1.00;
-
-use Getopt::Long();
-
-my $getopt = Getopt::Long::Parser->new(
-    config => [qw/gnu_getopt bundling_override no_ignore_case pass_through/]
+use ExtUtils::MakeMaker;
+our (%dev_requires, %runtime_suggests);
+(do './maint/Makefile.PL.include' or die $@) unless -f 'META.yml';
+
+my $eumm_version = eval $ExtUtils::MakeMaker::VERSION;
+my %eumm_args = (
+    NAME => 'DBIx::Class::Schema::Loader',
+    VERSION_FROM => 'lib/DBIx/Class/Schema/Loader.pm',
+    LICENSE => 'perl',
+    MIN_PERL_VERSION => '5.008001',
+    CONFIGURE_REQUIRES => {
+        'ExtUtils::MakeMaker' => '0',
+    },
+    TEST_REQUIRES => {
+        'DBD::SQLite'     => '1.29',
+        'Test::Exception' => '0.31',
+        'Test::More'      => '0.94',
+        'Test::Warn'      => '0.21',
+        'Test::Deep'      => '0.107',
+        'Test::Differences' => '0.60',
+        # temporary, needs to be thrown out
+        'DBIx::Class::IntrospectableM2M' => 0,
+        # core, but specific versions not available on older perls
+        'File::Temp'      => '0.16',
+        'File::Path'      => '2.07',
+    },
+    test => {TESTS => 't/*.t t/*/*.t'},
```

Comment:
This needs to include `t/*/*/*.t`, since the backcompat tests are two directories deep. There aren't actually any files matching `t/*/*.t`, but I guess it doesn't hurt to include it for future-proofing.

Reply:
Fixed. Counting is hard!